src/components/InputForm.jsx

```diff
@@ -0,0 +1,54 @@
+import { URL } from "../utils/api";
+
+export const InputForm = ({ setMessage, message, setThoughts }) => {
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    try {
+      const res = await fetch(URL, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ message: message }),
+      });
+
+      if (!res.ok) {
+        throw new Error(`HTTP error! status: ${res.status}`);
+      }
+
+      const data = await res.json();
+      setThoughts((prevThoughts) => [data, ...prevThoughts]);
+      setMessage("");
+    } catch (error) {
+      console.error("Error submitting thought:", error);
+    }
+  };
+
+  const MAX_CHARS = 140;
+
+  return (
+    <div className="thought-container input-form">
+      What's making you happy right now?
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          value={message}
+          onChange={(e) => setMessage(e.target.value)}
+          required
+        />
```

Comment:
Don't forget labels when working with form elements! And perhaps this should be a textarea since it's a multiline input?